README.md

```diff
@@ -1,4 +1,9 @@
-[![Build Status](https://ci.centos.org/buildStatus/icon?job=devtools-rh-che-build-che-credentials-master)](https://ci.centos.org/view/Devtools/job/devtools-rh-che-build-che-credentials-master/)
+| Last build        | Job           |
+| ------------- |:-------------:|
+| [![Build Status](https://ci.centos.org/buildStatus/icon?job=devtools-rh-che-build-che-credentials-master)](https://ci.centos.org/view/Devtools/job/devtools-rh-che-build-che-credentials-master/) | build che credentials |
+| [![Build Status](https://ci.codenvycorp.com/buildStatus/icon?job=rh-che-ci-master)](https://ci.codenvycorp.com/job/rh-che-ci-master/) | rh-che ci master |
+| [![Build Status](https://ci.centos.org/buildStatus/icon?job=devtools-rh-che-rh-che-compatibility-test-dev.rdu2c.fabric8.io)](https://ci.centos.org/view/Devtools/job/devtools-rh-che-rh-che-compatibility-test-dev.rdu2c.fabric8.io/) | compatibility check |
```

Comment:
and this one - "integration tests against upstream"

Reply:
I would suggest "integration tests against latest upstream SNAPSHOT" to be more concrete.